.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+### Summary of Changes
+
+<!---
+One sentence or bullet list of changes that were included in this PR. Example:
+- The changes in this PR grants ability to ... fix a bug that ... --->
+
+---
+
+### Summary of Implementation Logic
+
+<!---
+A short, bulleted list summarizing your implementation logic to help a reviewer understand at a high level of how your changes work. Example:
+
+- Creates new "BandService.js" file in "services" directory where API calls are executed
+- Imports BandService into SelectPlayers and calls bandService.getBands() method in useEffect() hook
+- Sets state in ChoosePlayers.tsx with fetched data, handles error with user-friendly error message
+
+Write your summary below this comment
+--->
+
+---
+
+### Issue Link
+
+<!--- Capture links to your ticket, and any related tickets, here --->
+
+Original Issue:
+
+Dependent Issues:
+
+Depends on Issues:
+
+---
+
+### Screen shots / Videos
+
+<!--- Place relevant screen shots and videos here that would help reviewers conceptualize the changes --->
+
+---
+
```

Comment:
Could we list one more thing: A/C (Acceptance Criteria). We should have this defined in the original issue and have a checklist. For example:
[ ] musician images are xxpx wide and xxpx tall
[ ] clicking on the button brings user to the correct page and /link
[ ] whatever the ticket is supposed to fix/bring into the code base

Reply:
Love the checklist. OK, so we'll make a Checklist for AC in the original issue, and then port it over into the PR so the checklist exists in both places?

Reply:
I think so! This will also force us (me...) to be more proactive when creating the original issue.

Reply:
Actually happened to me with the backend ticket--I forgot I included adding photos to the backend as an AC

Reply:
!OK added the section